src/App.js

```diff
@@ -1,4 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import './App.scss';
+import LandingPage from './Pages/LandingPage/LandingPage.jsx';
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import { createBrowserHistory } from 'history';
```

Comment:
the order of imports is faulty, should be something like this:

`
import { BrowserRouter as Router, Route } from "react-router-dom";
import { createBrowserHistory } from 'history';

import LandingPage from './Pages/LandingPage/LandingPage.jsx';

import './App.scss';
`